modules/label-categories-for-custom-metrics.adoc

```diff
@@ -0,0 +1,65 @@
+// module included in the following assemblies:
+//
+// * configuration/monitor-acs.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="label-categories-for-custom-metrics_{context}"]
+= Label categories for custom metrics
```

Comment:
I'd call them Metric categories, as there might be several metrics of `imageVulnerability` category with different set of labels, supported for this category.